app/models/role.rb

```diff
@@ -30,13 +30,25 @@ def organisations
     links["ordered_parent_organisations"]
   end
 
+  def current_holder
+    links["role_appointments"]
+      .select { |ra| ra.dig("details", "current") == true }
+      .first
```

Comment:
You can combine the calls to `.select({ ... }).first` [into a single call to `.find`](https://ruby-doc.org/core-2.6.5/Enumerable.html#method-i-find) (avoids unnecessarily performing the `select` block to the later elements in the array).

Reply:
Very true! 👍